### DISCUSSION TITLE
FluxStore Listing Booking

### DISCUSSION THREAD

**Discussion by kareemelmashad** [inspireui/support#5118] [New Feature/Ideas Requests]
FluxStore Listing, after booking done , u only get message , should be a button to take u see ur booking status there's bookings requires owner confirmations so you should give an option to user to see it , also some instant bookings requires payments once u book it give same message like thank u or confirmed while user didn't pay yet, it should give a button for payment after booking

**Comment by DavideM12** [comment]
Yes, it's true
